charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+---
+eol-exporter:
+  enabled: true
+  namespaceOverride: ''
+  applicationName: eol-exporter
+  externalSecrets:
+    enabled: false
+  deployment:
+    enabled: true
+    image:
+      repository: saritasallc/eol-exporter
+      tag: prod-b9828d4
+      digest: ''  # if set to a non empty value, digest takes precedence on the tag
+      pullPolicy: IfNotPresent
+    initContainers: []
+    command: ''
+    args: []
+    envFrom: {}
+    env:
+      ENVIRONMENT:
+        value: prod
```

Comment:
this should not be defined in the values of the chartt, instead - define the environment from the actual declaration (whete the chart is instantiated), not here

also why do you need ENVIRONMENT env var? What would change if it's not passed or passed a different value?

Reply:
explain the purpose for all ENV vars you define here in the env: section

Reply:
Added comments for each var.

ENVIRONMENT var removed, as it's not used